src/sdk/endpoint_stats_collector.js

```diff
@@ -48,6 +48,25 @@ class EndpointStatsCollector {
         }
     }
 
+    async _send_nsfs_stats() {
+        // LMLM Not sure if there is a point of retry sending it, We can send it in the next iteration.
+        // this.send_nsfs_stats = null;
+        // LMLM TODO: remove the log??
+        const _nsfs_stats = { nsfs_stats: { io_stats: this.nsfs_counters } };
+        dbg.log0(`LMLM _nsfs_stats: ${JSON.stringify(_nsfs_stats)}`);
```

Comment:
Clean or change all the LMLM prints and remarks, or follow on them for the next review cycles.